jwst/white_light/white_light.py

```diff
@@ -9,7 +9,7 @@
 log.setLevel(logging.DEBUG)
 
 
-def white_light(input):
+def white_light(input,min_wave=None,max_wave=None):
```

Comment:
Don't know if the flake8 tests will catch this, but spaces after commas are encouraged.

Also, an opportunity to update the docstring!

Reply:
Yeah, I was surprised that flake8 did not complain about several places where white space would normally be used.

Reply:
Hmmm, there might be some exclusion at work here. Wonder who would have set that up...